src/hooks/usePrices.ts

```diff
@@ -0,0 +1,46 @@
+import { BigNumber } from "@ethersproject/bignumber";
+import { formatUnits } from "@ethersproject/units";
+import { useQuery } from "react-query";
+import { queryAssertion } from "src/helpers";
+
+import { useOhmDaiReserveContract } from "./useContract";
+import { useCurrentIndex } from "./useCurrentIndex";
+
+export const ohmPriceQueryKey = () => ["useOhmPrice"];
+
+/**
+ * Returns the market price of OHM.
+ */
+export const useOhmPrice = () => {
+  const reserveContract = useOhmDaiReserveContract();
+
+  return useQuery<number, Error>(ohmPriceQueryKey(), async () => {
+    const [ohm, dai] = await reserveContract.getReserves();
+
+    return parseFloat(formatUnits(dai.div(ohm), 9));
```

Comment:
It would be nice if we put the decimal magic numbers in a constants file to know how they were determined

Reply:
Done